docs/docs/reference/contextual/extension-methods.md

```diff
@@ -21,9 +21,11 @@ circle.circumference
 ### Translation of Extension Methods
 
 Extension methods are methods that have a parameter clause in front of the defined
-identifier. They translate to methods where the leading parameter section is moved
-to after the defined identifier. So, the definition of `circumference` above translates
-to the plain method, and can also be invoked as such:
+identifier. They translate to functions where the leading parameter list is 
+either turned into the first argument list of the function or
+into the last argument list in case of a right-associative identifier.
```

Comment:
It's actually the second parameter list for a right associative identifier, not the last one. But I would not mention this here, since it is too confusing for the intro. I think the existing discussion in the operators subsection is OK, or maybe it can be expanded a bit further.

Reply:
removed the information about right associative identifier from this section